addons/web_editor/static/src/scss/web_editor.common.scss

```diff
@@ -990,3 +998,12 @@ section, .oe_img_bg, [data-oe-shape-data] {
         margin-bottom: 1rem;
     }
 }
+
+code.o_inline_code {
+    padding: .2em .4em;
+    border-radius: $border-radius;
+    border: 0;
+    background-color: $gray-200;
```

Comment:
Uff.. in dark-mode it matches the fromView background-color :/
| bright | dark |
|--------|--------|
| ![image](https://github.com/odoo/odoo/assets/8558351/3ad7455a-ba6d-4e3c-82de-dbea9d68d07c) | ![image](https://github.com/odoo/odoo/assets/8558351/43d961ce-1bdf-48a6-b40f-437d121859e1) | 

I'll let you investigate but seems a good scenario for CSS variables, something like:

```scss
background-color: var(--o-inline-code-background-color, #{$gray-200});
```

Reply:
Thanks for the feedback. Updated to: 

| bright | dark |
|--------|--------|
| ![image](https://github.com/odoo/odoo/assets/80678921/58738917-129c-42dc-aee8-00540b9651b6) | ![image](https://github.com/odoo/odoo/assets/80678921/cc397098-fda0-40b6-9dee-5cb463172b21) |